service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/disk/ControlledAzureDiskResource.java

```diff
@@ -82,6 +88,32 @@ public ControlledAzureDiskResource(DbResource dbResource) {
             .addParameter("diskName", getDiskName()));
   }
 
+  /** {@inheritDoc} */
+  @Override
+  public void addCreateSteps(
+      CreateControlledResourceFlight flight,
+      String petSaEmail,
+      AuthenticatedUserRequest userRequest,
+      FlightBeanBag flightBeanBag) {
+    RetryRule cloudRetry = RetryRules.cloud();
```

Comment:
Interesting. I had assumed these would return a `Step` or list of steps, rather than be void and take in the target flight. Might be more mockable/unit testable that way.